### PR TITLE
Add file include option to web update scripts

### LINKED ISSUE #148: Firebase snippets-web: Ignore specific folders
In the firebase/snippets-web repo, we need to freeze the Firebase version on specific folders (https://github.com/firebase/snippets-web/pull/231#issuecomment-909010451).

### PR DESCRIPTION
**Pull request by jhuleatt**
fixes #148 

- [x] Add an arg to the html, js, and node `latest-deps` scripts to allow them to only target certain folders, using a [pattern](https://www.gnu.org/software/findutils/manual/html_node/find_html/Shell-Pattern-Matching.html).
- [x] Update the Firebase kokoro scripts (`.kokoro/firebase/build-html`, `.kokoro/firebase/build-node`) to use the arg
- [x] Update the Firebase web config files (`.kokoro/firebase/snippets-web-html.cfg`, `.kokoro/firebase/snippets-web-node.cfg`) to set the correct arg

cc @kroikie

### CODE REVIEW

File: use-latest-deps-node.sh [hunk 1]
```diff
@@ -99,6 +105,12 @@ npm --prefix ../ install yarn@1.22.10
 
 # Find all package.json files.
 files=$(find . -name "package.json" -not -path "**/node_modules/*")
```
Comment (jhuleatt) [inline use-latest-deps-node.sh:107]: left this in by accident

Comment (jhuleatt) [reply use-latest-deps-node.sh]: fixed

File: use-latest-deps-node.sh [hunk 2]
```diff
@@ -16,17 +16,19 @@
 
 print_usage () {
   (>&2 echo "Usage:")
-  (>&2 echo "    $0 [-d] [-p regex] github-user/repository-name")
+  (>&2 echo "    $0 [-d] [-p regex] [-i pattern] github-user/repository-name")
   (>&2 echo "Arguments:")
   (>&2 echo "    -d: do a dry-run. Don't push or send a PR.")
   (>&2 echo "    -p: Only update Node packages that match the given regex")
+  (>&2 echo "    -i: Only include files whose path match the given pattern.")
```
[the remark this answers is not in the document]
Comment (jhuleatt) [reply use-latest-deps-node.sh]: added a comment with link to docs in each file

File: .kokoro/firebase/build-html.sh [hunk 3]
```diff
@@ -32,14 +32,29 @@ FIREBASE_SDK_VER=$(npm view firebase --json | jq -r '.["dist-tags"].latest')
...
+  ../use-latest-deps-js.sh "firebase/[0-9]*\.[0-9]*\.[0-9]*/" "firebase/${FIREBASE_SDK_VER}/" "${DPEBOT_REPO}"
+  # Updating CDN Firebase SDK dependencies.
+  ../use-latest-deps-html.sh "firebasejs/[0-9]*\.[0-9]*\.[0-9]*/" "firebasejs/${FIREBASE_SDK_VER}/" "${DPEBOT_REPO}"
+  ../use-latest-deps-js.sh "firebasejs/[0-9]*\.[0-9]*\.[0-9]*/" "firebasejs/${FIREBASE_SDK_VER}/" "${DPEBOT_REPO}"
+  # Updating CDN FirebaseUI dependencies.
+  ../use-latest-deps-html.sh "firebaseui/[0-9]*\.[0-9]*\.[0-9]*/" "firebaseui/${FIREBASEUI_VER}/" "${DPEBOT_REPO}"
+  )
+else
+  (
+  cd repo-to-update
```
Comment (tswast) [inline .kokoro/firebase/build-html.sh:49]: Is this `cd` redundant? I see it was moved about the `if` statement.

**Review by tswast — APPROVED** [review]
LGTM except for one nit

File: .kokoro/firebase/build-html.sh (hunk 3 above)
Comment (jhuleatt) [reply .kokoro/firebase/build-html.sh]: thanks for catching that, fixed!

**Comment by jhuleatt** [comment]
Thanks for reviewing, @tswast! Since I don't have merge access, can you please squash & merge?

**Pull request merged**